moveit_msgs/CollisionObject co
---
